<!DOCTYPE html>
<html lang="en">
  <head>
    <title>Lisa Levoir - M.S. Biostatistics</title>
    <meta charset="UTF-8" />
    <meta
      name="description"
      content="Lisa Levoir, MS Biostats, Strong Research Experience"
    />
    <meta
      name="keywords"
      content="Biostatistics, Biologist, Scientist, Researcher"
    />
    <meta author="Lisa Levoir" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <link rel="stylesheet" href="style.css" />
  </head>

  <body>
    <div class="nav-bar">
      <a href="#about">About</a>
      <a href="#experience">Experience</a>
      <a href="#publications">Publications</a>
      <a href="#contact">Contact</a>
    </div>
    <div class="section">
      <a class="anchor" id="about"></a>
      <div class="about-desc">
        <div class="img-headshot">
          <img
            width="512"
            height="512"
            src="/img/LisaLevoirphoto.jpeg"
            alt="A headshot of Lisa Levoir"
          />
        </div>
        <h1>Lisa Levoir</h1>
        <sub>MS in Biostatistics</sub>
        <p>
          Strong analytical problem solver and critical thinker interested in
          translational biological research.
        </p>
        <a class="button br-margin" href="https://linkedin.com/in/llevoir"
          >LinkedIn</a
        >
        <a class="button br-margin" href="mailto:lisa.m.levoir@vanderbilt.edu"
          >Email</a
        >
      </div>
    </div>

    <hr />
    <div class="section">
      <a class="anchor" id="experience"></a>
      <h2>Experience</h2>
      <div>
        <h3>Master's Degree in Biostatistics</h3>
        <sub>Vanderbilt University | Aug. 2022 - Present</sub>
        <p>
          Engaged in applied and theoretical coursework under 3 dominant
          statistical inference paradigms (Bayesian, Frequentist, and Maximum
          Likelihood) with experience in programming analysis in R and Stata.
        </p>
      </div>
      <div>
        <h3>Research Tech II - Lab Manager</h3>
        <sub>Fred Hutch | Oct. 2019 - Apr 2022</sub>
        <p>
          The Goo lab investigated the host-pathogen dynamics of flavivirus
          infections including dengue, Zika and West Nile virus. Key focuses
          include antibody mediated immunity and viral escape.<br />
          Contributions:
        </p>

        <ul>
          <li>
            Established the infrastructure for single B cell transcriptomics for
            antibody discovery in collaboration with postdoc
          </li>

          <li>
            Optimized pipeline for high-throughput antibody (IgG) production,
            purification, and characterization following selection via a
            Bayesian phylogenetic Markov Model
          </li>

          <li>
            Onboarded and trained rotation students (5) and summer undergraduate
            (1); contributed to their research projects
          </li>

          <li>
            Fostered a welcoming, productive, and collegial working environment
            for the newly established lab at Fred Hutch
          </li>

          <li>
            Published an invited commentary (Levoir & Goo, EBiomedicine 2020) on
            a relevant manuscript describing the use of single cell
            transcriptomics to profile the B cell repertoire following dengue
            virus infection.
          </li>
          <ul>
            <li>
              Contributed B cell receptor sequencing data which was included in
              a methods publication: Ralph, D. K., & Matsen IV, F. A. (2022).
              Inference of B cell clonal families using heavy/light chain
              pairing information. PLOS Computational Biology, 18(11), e1010723.
              <a href=" https://doi.org/10.1371/journal.pcbi.1010723"
                >https://doi.org/10.1371/journal.pcbi.1010723</a
              >
            </li>
          </ul>
        </ul>
      </div>
      <div>
        <h3>Research Technician - Clinical Trial Endpoints</h3>
        <sub>Fred Hutch | Jun. 2017 - Oct. 2019</sub>
        <p>HIV Vaccine Trials Network (HVTN) led by Dr. McElrath</p>
        <ul>

          <li>
            Developed procedures, drafted road maps for clinical studies,
            independently analyzed and interpreted data, and composed study
            reports to share findings.
          </li>

          <li>
            Employed functional cellular immunology and phenotyping assays to
            characterize immune responses to HIV, Ebola, and malaria vaccine
            candidates.
          </li>

		  <li>
            Consistently performed to high standards in a clinically regulated
            (GCLP) high-throughput team environment; met deadlines and
            prioritized work on multiple projects.
          </li>

          <li>
            Completed Activation Induced Marker (AIM) assay development for the
            detection of antigen-specific T cell responses. Responsible for
            procedure development, literature consultation, study plan writeup
            and analysis. Presented results at a UW CFAR (University of
            Washington Center for AIDS Research) conference.
          </li>
          <ul>
			<li>
				Contributed data to our inter-lab concordance study (between Seattle
				and Cape Town) which was publised: Dintwe, O. B., De Rosa, S. C.,
				Huang, Y., Flach, B. S., Manso, B., Carter, D., ... &
				Andersen-Nissen, E. (2022). Achieving intracellular cytokine
				staining assay concordance on two continents to assess HIV
				vaccine-induced T-cell responses. Journal of leukocyte biology,
				112(5), 1167-1181.
				<a href="https://doi.org/10.1002/JLB.5MA0522-668R"
				>https://doi.org/10.1002/JLB.5MA0522-668R</a
				>
			</li>
        </ul>
      </div>
      <div>
        <h3>
          Minnesota Department of Health Infectious Disease Labs - Virology
        </h3>
        <sub
          >Student Worker Paraprofessional Senior | Fall 2016 - June 2017</sub
        >
        <ul>
          <li>
            Validated new reagents for the detection of arbovirus infection in serum samples using IgM antibody capture enzyme-linked immunosorbent assay (MAC-ELISA). 
            Also participated in West Nile Virus surveillance in mosquito specimens.
          </li>
        </ul>
        <sub>Taylor Public Health Fellow | Summer 2016</sub>
        <ul>
          <li>
            One of 4 students to win grant funding from the Taylor Public Health
            Fellowship in Summer 2016
          </li>
          <li>
            Optimized a seminested reverse transcription PCR for the detection of enterovirus in clinical samples for the Severe Acute Respiratory Illness (SARI) public health surveillance initiative. Presented this project to the Infectious Disease Labs staff.
          </li>
        </ul>
      </div>
      <div>
        <h3>Summer Undergraduate Research Fellow</h3>
        <sub> Mayo Clinic | May 2015 - Aug. 2015</sub>
        <p>
          Researched how regulatory T cells interact with innate lymphoid cells and Th-17 cells to maintain gut homeostasis, and the role of IL-10 in this crosstalk. Presented this project to the Mayo Graduate School Immunology department. The project
          culminated in an undergraduate poster session and presentation to the
          Mayo Graduate School Immunology department.
        </p>
      </div>
    </div>

    <hr />
    <div class="section">
      <a class="anchor" id="publications"></a>
      <h2>Publications</h2>

      <ol>
        <li>
          Belmont, L., Contreras, M., Cartwright-Acar, C.H., Marceau, C.D., Agrawal, A., Levoir, L.M., Lubow, J. and Goo, L., 2024. Functional genomics screens reveal a role for TBC1D24 and SV2B in antibody-dependent enhancement of dengue virus infection. Journal of Virology. 
          bioRxiv 2024.04.26.591029; doi:
          <a href="https://doi.org/10.1101/2024.04.26.591029"
            >https://doi.org/10.1101/2024.04.26.591029</a
          >
        </li>
        <li>	
          Contreras, M., Stuart, J.B., Levoir, L.M., Belmont, L. and Goo, L., 2024. Defining the impact of flavivirus envelope protein glycosylation site mutations on sensitivity to broadly neutralizing antibodies. mBio, 15(2), pp.e03048-23. mBio15:e03048-23.<a
            href="https://doi.org/10.1128/mbio.03048-23"
            >https://doi.org/10.1128/mbio.03048-23</a
          >
        </li>
        <li>
          Lubow, J., Levoir, L.M., Ralph, D.K., Belmont, L., Contreras, M., Cartwright-Acar, C.H., Kikawa, C., Kannan, S., Davidson, E., Duran, V. and Rebellon-Sanchez, D.E., 2023. Single B cell transcriptomics identifies multiple isotypes of broadly neutralizing antibodies against flaviviruses. PLOS Pathogens, 19(10), p.e1011722.<a 
          <a href="https://doi.org/10.1371/journal.ppat.1011722"
          >https://doi.org/10.1371/journal.ppat.1011722</a
          >
        </li>
        <li>
          Kikawa, C., Cartwright-Acar, C.H., Stuart, J.B., Contreras, M., Levoir, L.M., Evans, M.J., Bloom, J.D. and Goo, L., 2023. The effect of single mutations in Zika virus envelope on escape from broadly neutralizing antibodies. Journal of Virology, 97(11), pp.e01414-23.
          <a
          href="https://doi.org/10.1128/jvi.01414-23"
            >https://doi.org/10.1128/jvi.01414-23</a
          >
        </li>
        <li>
          Levoir, L. and Goo, L., 2020. A (class-) switch in the antibody response may distinguish primary from secondary dengue virus infection. EBioMedicine, 56.
          <a
            href="https://doi.org/10.1016/j.ebiom.2020.102790"
            >https://doi.org/10.1016/j.ebiom.2020.102790</a
          >
        </li>
      </ol>
    </div>
    <hr />
    <div class="footer">
      <a class="anchor" id="contact"></a>
      <h2>Contact</h2>
      <a class="button br-margin" href="mailto:lisa.m.levoir@vanderbilt.edu"
        >lisa.m.levoir@vanderbilt.edu</a
      >
      <a class="button br-margin" href="https://linkedin.com/in/llevoir"
        >linkedin.com/in/llevoir</a
      >
    </div>
  </body>
</html>
